src/CMakeLists.txt

```
@@ -109,6 +110,29 @@ endif()
 #set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=hidden")
 
+# Properly set the rpath in a cross-platform way
+if(APPLE)
+  set(CMAKE_MACOSX_RPATH ON)
+  set(_rpath_portable_origin "@loader_path")
+else()
+  set(_rpath_portable_origin $ORIGIN)
+endif(APPLE)
+set(CMAKE_SKIP_BUILD_RPATH FALSE)
+if(NOT BUILD_INTREE)
+  set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
+  set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+  #_magnum.so needs to look at habitat_sim/_ext for its corrade dependencies
+  set(
+    CMAKE_INSTALL_RPATH
+    "${_rpath_portable_origin}:${_rpath_portable_origin}/habitat_sim/_ext/:../../../.."
```

```suggestion
    "${_rpath_portable_origin}"
    "${_rpath_portable_origin}/habitat_sim/_ext/"
    "../../../.."
```

This should be a semicolon-separated list (https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_RPATH.html#variable:CMAKE_INSTALL_RPATH).  Or giving multiple args to set will alleviate the need for a separator.

I think this should work with this changed

---

Also, the last one should probably be `"${_rpath_portable_origin}/../../../.."`?